src/go/k8s/go.mod

```diff
@@ -25,41 +25,55 @@ require (
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.74.0
 	github.com/prometheus/client_golang v1.20.0
 	github.com/prometheus/common v0.55.0
-	github.com/redpanda-data/common-go/rpadmin v0.1.5-0.20240814205445-8e6eb5806561
+	github.com/redpanda-data/common-go/rpadmin v0.1.7-0.20240909210234-44a1212753d7
 	github.com/redpanda-data/console/backend v0.0.0-20240303221210-05d5d9e85f20
 	github.com/redpanda-data/helm-charts v0.0.0-20240911060052-2bf9dd6f0996
-	github.com/redpanda-data/redpanda/src/go/rpk v0.0.0-20240105044330-c094966ca0cf
+	github.com/redpanda-data/redpanda/src/go/rpk v0.0.0-20240827155712-244863ea0ae8
+	github.com/spf13/afero v1.11.0
 	github.com/spf13/cobra v1.8.1
 	github.com/stretchr/testify v1.9.0
-	github.com/testcontainers/testcontainers-go/modules/redpanda v0.32.0
-	github.com/twmb/franz-go v1.16.1
-	github.com/twmb/franz-go/pkg/kadm v1.11.0
-	github.com/twmb/franz-go/pkg/kmsg v1.7.0
+	github.com/testcontainers/testcontainers-go v0.33.0
+	github.com/testcontainers/testcontainers-go/modules/redpanda v0.33.0
```

Comment:
Something broke with go dependency update. https://github.com/redpanda-data/redpanda-operator/pull/231#issuecomment-2353796886

Could you downgrade some of the dependency to see if that helps?